runners/dmn-tck-runner-jdmn/pom.xml

```diff
@@ -57,7 +57,7 @@
         <dependency>
             <groupId>com.goldmansachs.jdmn</groupId>
             <artifactId>jdmn-core</artifactId>
-            <version>1.3.0</version>
+            <version>2.0.1-SNAPSHOT</version>
```

Comment:
@opatrascoiu can you please double check if this is correct: on the runner you are using the 2.0.1-SNAPSHOT version, but on the properties file for the website report it says "2.1.0".